burr/core/persistence.py

```diff
@@ -54,6 +54,10 @@ def initialize(self):
         """Initializes the app for saving, set up any databases, etc.. you want to here."""
         pass
 
+    def is_initialized(self) -> bool:
```

Comment:
Oh, I see, we're catching the `NotImplemented`  as before...

Might be simpler just to have this default to `True`, then allow you to overwrite it.

Reply:
No because that leads to edge cases with backwards compatibility.